sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing Guide
+
+The Azure Communication Services SDK for Python currently consists of 4 different packages. While each package has its own set of environment variables to make their tests run successfully, all of them follow a similar structure that allows a smooth onboarding process.
+
+Let's get started with how to setup the repo itself.
+
+## Installation process
+
+To get started with any of the packages, change directory to the package you want to install and run the `pip install .` command. This will install all of the local files necessary for you to run the corresponding tests. It's important to note that if you made changes to the local files and want to run the tests again, you must run the `pip install .` command from the package root folder to update the files with your new changes.
+
+Once the package has been installed on your machine, let's jump on how to run the tests to see that everything is in order.
+
+## Testing
+
+When you go inside the tests folder of the package you are working with, you will see a folder called `recordings`. This folder contains, as its name suggests, recordings of successful calls to the API that allow us to run the tests in PLAYBACK mode and remove the necessity of hitting the actual resources every time we may want to test.
+
+### PLAYBACK
+
+To run the tests in PLAYBACK mode, set an environment variable called `AZURE_TEST_RUN_LIVE` and set its value to `false` (If the variable if not set, the default will be `false`). After your variable has been set, change directory to the `tests` folder of the package you're working on and run the `pytest .` command.
+
+If the tests are successful, we can proceed to run the tests in LIVE mode.
+
+### LIVE
```

Comment:
I think Sms live test need a env var `AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER` should we mention that somewhere in this doc?

Reply:
I mention about these special environment variables in line 27